index.html

```diff
@@ -119,15 +133,13 @@ <h3 class="title-subheader">Aug 23-26  //  Portland, OR </h3>
 		</nav>
 	</header>
 
-<!-- ## TABLET & DESKTOP NAV ### -->
-<!-- coming soon! -->
 
 <!-- ### SITE NAV END ### -->
 
 <!-- ## SCHEDULE SECTION ## -->
-<section class="schedule-section">
+<section class="schedule-section" id="schedule">
   <div class="qcTabTitle qcContainer clearfix">
-    <div id="schedule" class="col-4 col">
+    <div class="col-4 col">
       <h2 class="section-title">SCHEDULE</h2>
```

Comment:
on mobile the Schedule and Sponsors sections have extra height above the section title (which is not the case for what's deployed to production). i'm not sure what exactly might be causing that... maybe we can ship this and create a trello card to fix the styling bugs